index.d.ts

```diff
@@ -1819,7 +1819,7 @@ interface TableVirtualBodyProps extends PaneProps {
 export interface TableProps extends PaneProps {
 }
 
-export class Table extends React.PureComponent<TableProps> {
+export class Table extends React.FC<TableProps> {
```

Comment:
```
declare const Table: React.FC<TableProps> & {
     static members here
}
```

Reply:
I can make this change once all the other type changes are merged